doc/source/whatsnew/v1.1.0.rst

```diff
@@ -120,7 +120,7 @@ MultiIndex
 
 I/O
 ^^^
-
+- When writing directly to a sqlite connection :func:`to_sql` now supports the ``multi`` method (:issue:`29921`)
```

Comment:
Final comment: can you move this bullet point to the "Otherr Enhancements" section?

Reply:
Done